self-core-impl/src/main/java/com/selfxdsd/core/GithubIssueComments.java

```diff
@@ -28,24 +34,49 @@
 
     /**
      * Ctor.
+     *
      * @param issueUri Comments Issue URI.
      * @param resources Github's JSON Resources.
      */
     public GithubIssueComments(final URI issueUri,
                                final JsonResources resources) {
-        this.commentsUri = issueUri.resolve("comments");
+        final String issueUriStr = issueUri.toString();
+        String slash = "/";
+        if(issueUriStr.endsWith("/")){
+            slash = "";
+        }
+        this.commentsUri = URI.create(issueUriStr + slash + "comments/");
         this.resources = resources;
     }
 
     @Override
-    public Comment post(final String body) {
-        //POST /repos/:owner/:repo/issues/:issue_number/comments
-        throw new UnsupportedOperationException("Not implemented yet");
+    public Comment post(final String body, final String accessToken) {
+        final Resource resource = resources.post(
+                this.commentsUri,
+                Json.createObjectBuilder().add("body", body).build(),
+                accessToken
+        );
+        if (resource.statusCode() == 201) {
+            return new GithubComment(resource.asJsonObject());
+        } else {
+            throw new IllegalStateException("Github Issue Comment "
+                    + "was not created");
+        }
     }
 
     @Override
     public Iterator<Comment> iterator() {
-        //GET /repos/:owner/:repo/issues/:issue_number/comments
-        throw new UnsupportedOperationException("Not implemented yet");
+        final Resource resource = resources.get(this.commentsUri);
+        List<Comment> comments;
+        if (resource.statusCode() == 200) {
```

Comment:
@criske Can you please use ``HttpUrlConnection.STATUS_OK`` instead of hardcoding ``200``?